content/src/service/ServiceImpl.ts

```diff
@@ -38,6 +38,8 @@ export class ServiceImpl implements MetaverseContentService {
   private static LOGGER: ILoggerComponent.ILogger
   private readonly pointersBeingDeployed: Map<EntityType, Set<Pointer>> = new Map()
 
+  private readonly deploymentsCache: { cache: NodeCache; maxSize: number }
```

Comment:
Is this the rate limit cache? Why do you need to change that here? Are these changes compatible with the ones in https://github.com/decentraland/catalyst/pull/917 ?

Reply:
I just removed the `ServiceFactory` so I moved some logic to the constructor here, will catch up with #917 changes.